### DISCUSSION TITLE
Why import maps (&& ! deps.ts)?

### DISCUSSION THREAD

**Discussion by NfNitLoop** [freshframework/fresh#256] [General]
Just found this bug:  https://github.com/lucacasonato/fresh/issues/255

I'm curious -- why does Fresh use import maps instead of the common [deps.ts] method I've seen for declaring dependencies?

[deps.ts]: https://deno.land/manual/examples/manage_dependencies

**Comment by lucacasonato** [comment]
Because `deps.ts` forces all dependencies through a single file, any file which imports anything from `deps.ts` will transitively import everything from `deps.ts`. This bloats client side bundles. With an import map you can more easily be selective about what you load.
